reference/api/settings.mdx

```diff
@@ -2695,6 +2719,26 @@ This option can be useful when working with large Meilisearch projects. Consider
 **Activating `binaryQuantized` is irreversible.** Once enabled, Meilisearch converts all vectors and discards all vector data that does fit within 1-bit. The only way to recover the vectors' original values is to re-vectorize the whole index in a new embedder.
 </Capsule>
 
+##### `indexingEmbedder` and `searchEmbedder` <NoticeTag type="experimental" label="experimental" />
+
+When using a [composite embedder](#composite-embedders), configure separate embedders Meilisearch should use when vectorizing documents and search queries.
+
+`indexingEmbedder` often benefits from the higher bandwidth and speed of remote providers so it can vectorize large batches of documents quickly. `searchEmbedder` may often benefits from the lower latency of processing queries locally.
+
+Both fields must be an object and accept the same fields as a regular embedder, with the following exceptions:
+
+- `indexingEmbedder` and `searchEmbedder` must use the same `model`
```

Comment:
this is not strictly true. At the abstract level both embedders should use the same model, but regarding the `model` parameter, if one sub-embedder is a `huggingFace` (local) embedder, and the other is a `rest` embedder  that calls a remote providers, `model` will not even be an available parameter for the REST embedder.